src/IotCentral/IotCentral.Test/IotCentral.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.IotCentral" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.IotCentral" Version="2.1.0" />
```

Comment:
to use the latest one (https://www.nuget.org/packages/Microsoft.Azure.Management.IotCentral/)